### PR TITLE
[LW 9606] Add dark mode to stories

### PR DESCRIPTION
**Pull request by lucas-barros**
# Checklist

- [x] JIRA - https://input-output.atlassian.net/browse/LW-9606
- [ ] ~~Proper tests implemented~~
- [ ] ~~Screenshots added.~~

---

## Proposed solution

Create a decorator to add dark mode to stories and add it as default to global decorators
Add flag to disable global theme decorator to cover dialog/modal stories, as those need separated dark mode story

## Testing

Check: https://659e90c5edb661a3f34725d8-pmombxtshr.chromatic.com/

### CODE REVIEW

**Review by VanessaPC — APPROVED** [review]
LGTM!

**Review by DominikGuzei — APPROVED** [review]
Great work 👏

File: packages/core/.storybook/preview.js [hunk 1]
```diff
@@ -19,9 +19,16 @@ export const preview = {
...
 
 export const decorators = [
-  (Story) => (
-    <ThemeProvider colorScheme={ThemeColorScheme.Light}>
-      <Story />
-    </ThemeProvider>
-  )
+  (Story, args) => {
+    const { decorators: { colorSchema = true } = {} } = args.parameters;
+    return colorSchema ? colorSchemaDecorator(Story, args) : <Story />;
```
[the remark this answers is not in the document]
Comment (lucas-barros) [reply packages/core/.storybook/preview.js:24]: Yes, colorSchemaDecorator will display the same story side by side with light/dark theme. "theme" is to add just one theme to a single story for cases where "localTheme" does not work like modals, dialogs, selects.

**Review by renanvalentin — APPROVED** [review]
LGTM!

**Pull request merged**